### ISSUE TITLE
Add Starlite to the benchmarks

### ISSUE THREAD

**Issue by metaperl** [klen/py-frameworks-bench#191]
Starlite is gaining in popularity due to it's appealing featureset. Please [include it](https://github.com/starlite-api/starlite).

**Comment by patx** [comment]
Agreed. Litestar should be added forsure. Maybe also [MicroPie](https://patx.github.io/micropie).
